criteria/archival.md

```diff
@@ -0,0 +1,57 @@
+---
+order: 15
+---
+
+# Archive for the long-term
+
+## Requirements
+
+* You MUST have a copy of your codebase in a long-term archive
+* You MUST not rely on your regular development platform as long-term archive,
+  unless it *explicitly* provides long-term archival guarantees
+* You SHOULD archive both source code itself and all associated development
+  artifacts (version control history, issues, discussions, etc.)
+* You SHOULD archive your codebase frequently, ideally at every change
```

Comment:
"You SHOULD archive your codebase frequently, ideally at every change" seems misleading to me.

Maybe "Each deployed version of a codebase SHOULD be archived" or similar wording?